File: compiled_lang/cs/f13.loop.cs
// testbox: title="Enumerable.Range().Reverse() with foreach"
using System;
using System.Linq;

class F13Loop
{
    static void Main()
    {
        foreach (int count in Enumerable.Range(1, 10).Reverse())
            Console.WriteLine($"Count is {count}");
    }
}

File: compiled_lang/go/f13.loop.go
// testbox: title="range-over-int (Go 1.22+)"

package main

import "fmt"

func main() {
	// range-over-int counts 0..9; map it back to 10..1
	for i := range 10 {
		fmt.Println("Count is", 10-i)
	}
}
